lib/beaker/host.rb

```diff
@@ -504,13 +530,13 @@ def do_rsync_to from_path, to_path, opts = {}
   end
 
   [
-    'unix',
-    'aix',
-    'mac',
-    'freebsd',
-    'windows',
-    'pswindows',
-    'eos',
+      'unix',
+      'aix',
+      'mac',
+      'freebsd',
+      'windows',
+      'pswindows',
+      'eos',
```

Comment:
our old spacing is what we want to use here, since we use 2 spaces rather than 4 for indenting.  If there's some other reason to do this that I'm not seeing, please let me know, and we can discuss this further.